### DISCUSSION TITLE
Navigation Button when no Route loaded

### DISCUSSION THREAD

**Discussion by MotoUKRider** [devemux86/cruiser#121] [Q&A] [answered]
The Navigation button on the main app screen, when no route is loaded, does nothing except "tell you off" when you press it.

Idea: When pressed - and only when there is no route loaded - invoke the Import/Load Route dialog so you can pick a GPX/ITN/Kurviger file in the usual way.  3 presses become 1 :)

**Comment by devemux86** [comment, marked as answer]
@MotoUKRider thanks for the suggestion!

There is also a similar discussion here:
- https://github.com/devemux86/cruiser/discussions/64

**Reply by MotoUKRider** [reply]
Ha! Sometimes I forget just how much I've forgotten :)

**Reply by devemux86** [reply]
No problem, you also created both topics!
Let's discuss it in the other older topic where some users have already shown their preference. :+1:
